packages/@o3r-training/showcase-sdk/eslint.local.config.mjs

```diff
@@ -13,14 +13,10 @@ export default [
   {
     name: '@o3r-training/showcase-sdk/ignores',
     ignores: [
-      'src/api',
-      'src/models/base',
-      'src/spec/api-mocks.ts',
-      'src/spec/operation-adapter.ts',
-      'dist',
```

Comment:
I assume this is already ignored?

Reply:
yes already ignored in the shared config at the root of the monorepo